basic/src/lib.rs

```diff
@@ -100,7 +100,7 @@ pub struct BasicMachine {
     #[chip]
     output: OutputChip,
     #[chip]
-    range: RangeCheckerChip, // TODO: Specify 8-bit RC chip
+    range: RangeCheckerChip<256>,
```

Comment:
I was hoping to rename this `range_256` or `range_8_bit`, as I think we'll have others later, but the derive's assumption about matching method names was getting in the way. We can leave the name for now but I think we may need to remove that assumption at some point.